gazelle/internal/swift/is_builtin_module_test.go

```diff
@@ -21,7 +21,22 @@ func TestIsBuiltInModule(t *testing.T) {
 		{msg: "does not exist", name: "DoesNotExist", exp: false},
 	}
 	for _, tt := range tests {
-		actual := swift.IsBuiltInModule(tt.name)
+		actual := swift.IsBuiltInFramework(tt.name)
+		assert.Equal(t, tt.exp, actual, tt.msg)
+	}
+}
+
+func TestIsBuiltInSwiftModule(t *testing.T) {
+	tests := []struct {
+		msg  string
+		name string
+		exp  bool
+	}{
+		{msg: "os", name: "os", exp: true},
+		{msg: "does not exist", name: "DoesNotExist", exp: false},
+	}
```

Comment:
Could we add one `true` test case?

Reply:
"os" is the true case but let me know if you mean another one

Reply:
That's the best time to do them though! I'll get this rebased and fixed soon